src/components/PopupWithImage.js

```diff
@@ -1,43 +1,25 @@
-import Popup from "./PopUp.js";
-
+import Popup from "./Popup.js";
 
 export default class PopUpWithImage extends Popup {
-
-  constructor({ name, link }, popupSelector) {
-
-    super(popupSelector);
-
-    /* this._name = name;
-
-    this._link = link; */
+  constructor( popupSelector) {
+    super(popupSelector,);
```

Comment:
> It looks like when I delete this line, my cards are unable to be created because the other variables become undefined. Do i need to add this to the constructor and super selector?

There is a mistake in line 5 

It should be 


```js

    super({ popupSelector });

```


And now you should delete line 7